core/pkg/monitor/monitor.go

```diff
@@ -72,6 +73,21 @@ func makeMetadataRecord(metadata *service.MetadataRequest) *service.Record {
 	}
 }
 
+func getSlurmEnvVars() map[string]string {
```

Comment:
does it make sense to have this as another asset that only have probing?

Reply:
great idea! will do in the next pr